app/views/merchants/show.html.erb

```diff
@@ -12,6 +12,7 @@
   <%  @merchant.invoices_to_ship.each do |invoice_item| %>
       <p><%= invoice_item.item.name %></p>
       <p><%= link_to "Invoice ##{invoice_item.invoice.id}", "/merchants/#{@merchant.id}/invoices/#{invoice_item.invoice.id}" %></p> 
+      <p><%= invoice_item.created_at.strftime('%A, %B%e, %Y') %>
```

Comment:
this is looking for the "the date the invoice was created" -- consider changing to `invoice_item.invoice.created_at.strftime(...)`,   also missing an ending </p> on line 15